metpy/cbook.py

```diff
@@ -39,4 +37,103 @@ def get_test_data(fname, as_file_obj=True):
     return path
 
 
-__all__ = ('get_test_data', 'is_string_like', 'iterable')
+class CacheData(object):
+    def __init__(self):
+        self.cache_home = self.get_cache_location()
+        self.cache_version = 1
+
+        # Check if there is a cache version file. If not, this is a new cache, so create it.
+        cache_version_file_path = os.path.join(self.cache_home, 'cache_version.txt')
+        if not os.path.exists(cache_version_file_path ):
```

Comment:
If the file doesn't exist, we should probably just invoke the code to re-initialize the cache (which should probably be its own method). You could accomplish that by initializing `current_cache_version` to 0, and only reading the current version if the file exists.